Gruntfile.js

```diff
@@ -217,6 +228,10 @@ var config = require('./core/server/config'),
                         if (!filter) {
                             grunt.log.write(chunk);
                         }
+
+                        if (chunk.indexOf('Build successful') !== -1) {
```

Comment:
Is it worth using a regex here? e.g. /successful/ig?
To avoid if ember changes the output, the chance that this breaks is a little lower?

Reply:
Okay we can leave as is. I was just curious :)